src/renderer/components/mulmo_viewer/tabs/slide_tab.vue

```
@@ -46,7 +46,7 @@
           </div>
           <Button @click="increase" variant="outline">{{ t("ui.common.increase") }}</Button>
         </div>
-        {{ mulmoMultiLinguals?.[currentBeatId]?.["multiLingualTexts"]?.[currentLanguage]?.text || currentBeat?.text }}
+        {{ isScriptLang ? currentBeat?.text : (mulmoMultiLinguals?.[currentBeatId]?.["multiLingualTexts"]?.[currentLanguage]?.text ?? t("ui.common.noLang"))}}
```

ESLint

[error] Replace `·isScriptLang·?·currentBeat?.text·:·(mulmoMultiLinguals?.[currentBeatId]?.["multiLingualTexts"]?.[currentLanguage]?.text·??·t("ui.common.noLang"))` with `⏎··········isScriptLang⏎············?·currentBeat?.text⏎············:·(mulmoMultiLinguals?.[currentBeatId]?.["multiLingualTexts"]?.[currentLanguage]?.text·??⏎··············t("ui.common.noLang"))⏎········`

(prettier/prettier)